Release/src/utilities/asyncrt_utils.cpp

```diff
@@ -991,7 +993,7 @@ datetime __cdecl datetime::from_string(const utility::string_t& dateString, date
 {
     datetime result;
     int64_t secondsSince1900;
```

Comment:
Might be worth initializing to 0

Reply:
I explicitly don't want to do that so that if there's any path where it isn't assigned the compiler will warn.